src/routes/home/HomeView.vue

```diff
@@ -55,7 +55,7 @@
               </v-row>
             </div>
             <v-window v-else v-model="tab" class="pa-4 overflow-y-auto">
-              <v-window-item :value="TABS.PLAY">
+              <v-window-item>
```

Comment:
By itself I wouldn't expect this to work, as I don't see anything that relatest he route `/spectate-list` to telling this component which list to render. To check, we should update the test

```
it('Does not show open or completed games in spectate tab')
```
in `home.spec` so that on line 306, instead of `cy.visit('/')`  and `cy.get('[data-cy-game-list-selector=spectate]').click();`, we call `cy.reload()` then confirm we are on the same list of games (by checking which tab is highlighted).

Likewise on the next test 
```
it('Shows ongoing games as available to spectate when user navigates to home page')
```
we should add a `cy.reload()` after navigating to the list of spectatable games to confirm that we stay on the list of spectatable games and can still join one.